src/lib/gameService.ts

```diff
@@ -12,41 +13,172 @@ export class GameService {
 			code: this.generateCode(),
 			currentPlayer: undefined,
 			players: [],
-			currentBid: undefined
+			currentBid: undefined,
+			maxPlayers: 8,
```

Comment:
We might need to have a chat about this - the current designs are for three-player games as anything beyond that doesn't really fit on the mobile screen. If you're keen to support more players, we should rework the game dtos to be more than 'left opponent' and 'right opponent'